package/yast2-bootloader.changes

```diff
@@ -1,9 +1,36 @@
 -------------------------------------------------------------------
+Wed Jul  5 12:27:37 CEST 2017 - shundhammer@suse.de
+
+- Merged master to storage-ng branch
+- 3.4.0
+
+-------------------------------------------------------------------
 Tue Jul  4 15:27:27 UTC 2017 - igonzalezsosa@suse.com
 
 - Adapt the module to use the new storage-ng during system cloning
   (bsc#1047245)
 - 3.3.9
+-------------------------------------------------------------------
+Fri Jun 23 09:14:05 CEST 2017 - schubi@suse.de
+
+- AuotYaST configuration: Set StorageDevices flag disks_valid
+  to true. So libstorage can scan valid disks and "/" partition
+  can be found. (bnc#1043132)
+- 3.3.0
```

Comment:
Please, remove the 3.2.21, 3.2.22 and 3.3.0 version numbers, as we have done in other sync merges